common/app/routes/Challenges/Output.jsx

```diff
@@ -13,19 +16,34 @@ const defaultOptions = {
   readOnly: 'nocursor'
 };
 
+const mapStateToProps = createSelector(
```

Comment:
There is no need to add a layer of memoization if there is only one value being selected for. In these situations it is fine just to use the selector (in this case the theme selector) with a plain function.